zkvm-host/bin/agg.rs

```diff
@@ -17,12 +18,9 @@ struct Args {
     #[arg(short, long, num_args = 1.., value_delimiter = ',')]
     proofs: Vec<String>,
 
+    /// L1 Head
     #[arg(short, long)]
-    multi: bool,
-
-    /// Verbosity level.
-    #[arg(short, long, default_value = "0")]
-    verbosity: u8,
+    latest_l1_head: B256,
```

Comment:
nit: Call this `latest_checkpoint_head` (match the terminology in the other file).